cpp/cmake/cudf-build-config.cmake.in

```diff
@@ -38,9 +56,34 @@ set(CUDF_MIN_VERSION_rmm "${CUDF_VERSION_MAJOR}.${CUDF_VERSION_MINOR}")
 include(@CUDF_SOURCE_DIR@/cmake/thirdparty/CUDF_GetRMM.cmake)
 # find gtest
 include(@CUDF_SOURCE_DIR@/cmake/thirdparty/CUDF_GetGTest.cmake)
+# find GoogleBench
+CPMFindPackage(NAME benchmark
+    VERSION         1.5.2
+    GIT_REPOSITORY  https://github.com/google/benchmark.git
+    GIT_TAG         v1.5.2
+    GIT_SHALLOW     TRUE
+    OPTIONS         "BENCHMARK_ENABLE_TESTING OFF"
+                    "BENCHMARK_ENABLE_INSTALL OFF")
+
+function(add_cudf_library_alias alias target)
```

Comment:
Should name this `_add_cudf_library_alias` to match `cudf-config.cmake.in` and later on we can hoist it out to a separate module we install.